### PR TITLE
Input and Output Files Print Full Paths

### PR DESCRIPTION
**Pull request by RoseSheehan**
Changed input and output files from printing name of file to printing location. Also updated expected output in test to expect file location.

### CODE REVIEW

File: tests/testthat/test_provSummarize.R [hunk 1]
```diff
@@ -9,6 +9,7 @@ expect_known_output(prov.summarize.file(test.data, save = FALSE, create.zip = FA
 # Test prov.summarize.run
 test.script <- system.file("testscripts", "warnings.r", package = "provSummarizeR", mustWork=TRUE)
 test.expected <- system.file("testsummaries", "warnings.expected", package = "provSummarizeR", mustWork=TRUE)
+prov.summarize.run(test.script) # REMOVE AT END
```
Comment (RoseSheehan) [inline tests/testthat/test_provSummarize.R:12]: This was for testing purposes only - please do not accept.

Comment (blernermhc) [reply tests/testthat/test_provSummarize.R:12]: I am confused.  Should this line not be here?  If that is the case, I think you need to remove the line and commit again.

**Comment by blernermhc** [comment]
I see that the Travis tests are failing.  Do they pass when you run them on your computer?

**Comment by RoseSheehan** [comment]
I've run into a new issue with this - will close the pull request for now and reopen when fixed. Apologies!

**Pull request closed without merging**